doc/Language/variables.rakudoc

```diff
@@ -1370,6 +1370,30 @@ all available resource files in a distribution by printing or by using other
 ways to inspect its value. Instead, use the API described above to
 access particular files.
 
+Since the C<%?RESOURCES> variable is only accessible as a compiled object,
```

Comment:
I'm not sure this first sentence is entirely accurate, as its a little more nuanced than that. For instance there are other compile time variables that can be used during runtime (such as `$?FILE`) inside of an e.g. test.

Maybe it would be better to say something like `%?RESOURCES is only accessible inside of modules. If you want to access %?RESOURCES outside of a module you'll need to expose that API yourself. One way to do that is to create a routine in the C<lib> directory to
return its value: ...`

Reply:
I'll work on changes tomorrow. Thanks.